api/src/main/resources/cb_review_api.yaml

```diff
@@ -737,6 +738,7 @@ definitions:
         description: The Value of the annotation if the AnnotationType is DATE
       annotationValueBoolean:
         type: boolean
+        default: false
```

Comment:
This should also default to null for the same reasons as the above comment.

Reply:
ok cool.